### PR TITLE
Features/search ui update

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

File: app/services/search-service.js [hunk 1]
```diff
@@ -85,6 +90,12 @@ define(['app', 'underscore', './local-storage-service', './app-config-service',
                         'group.sort': localizeFields(searchQuery.groupSort)
                     };
 
+                    if(searchQuery.highlight){
+                        queryGroupParameters.hl      = true;
+                        queryGroupParameters['hl.snippets'] = 5;
+                        queryGroupParameters['hl.fl']= searchQuery.highlightFields||'text_EN_txt';
```
Comment (sbilodeau) [inline app/services/search-service.js:96]: no Alignement 😱

File: app/views/search/search-filters/left-side-filter.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿define(['app', 'text!views/search/search-filters/left-side-filter.html', 'lodash', 'ngDialog',
     'components/scbd-angularjs-services/services/utilities'], function (app, template, _) {
 
-        app.directive('leftSideFilter', ['ngDialog', 'Thesaurus', 'locale', function (ngDialog, thesaurus, locale) {
+        app.directive('leftSideFilter', ['ngDialog', 'Thesaurus', 'locale', '$timeout', function (ngDialog, thesaurus, locale, $timeout) {
```
Comment (sbilodeau) [inline app/views/search/search-filters/left-side-filter.js:4]: $timoute unused

File: app/views/search/search-filters/left-side-filter.js [hunk 3]
```diff
@@ -35,7 +35,7 @@
 
                     $scope.showFilterDialog = function (schema, filter, facets) {
                         ngDialog.open({
-                            template: 'filtersDialog', height:'100%',
+                            template: 'filtersDialog', 
                             className: 'search-filters ngdialog-theme-default wide',
                             controller: ['$scope', '$timeout', 'thesaurusService', 'searchService', function ($scope, $timeout, thesaurusService, searchService) {
```
Comment (sbilodeau) [inline app/views/search/search-filters/left-side-filter.js:40]: $timoute unused

**Comment by blaisefonseca** [comment]
> can you drop refenrence to $timout in app/views/search/search-filters/left-side-filter.js

Done!

**Pull request merged**